cdap-api-common/src/main/java/co/cask/cdap/api/data/format/FormatSpecification.java

```diff
@@ -37,16 +38,21 @@ private FormatSpecification() {
     this(null, null, Collections.<String, String>emptyMap());
   }
 
-  public FormatSpecification(String name, Schema schema, Map<String, String> settings) {
+  public FormatSpecification(String name, @Nullable Schema schema, @Nullable Map<String, String> settings) {
     this.name = name;
     this.schema = schema;
     this.settings = settings == null ? Collections.<String, String>emptyMap() : settings;
   }
 
+  public FormatSpecification(String name, Schema schema) {
```

Comment:
So does this `Schema` also be `@Nullable`? Seems awkward that one constructor is nullable while the other is not.